src/Elasticsearch.Net/Configuration/Security/ApiKeyAuthenticationCredentials.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Security;
+
+namespace Elasticsearch.Net
+{
+	/// <summary>
+	/// Credentials for Api Key Authentication
+	/// </summary>
+	public class ApiKeyAuthenticationCredentials : IDisposable
+	{
+		public ApiKeyAuthenticationCredentials()
+		{
+		}
+
+		public ApiKeyAuthenticationCredentials(string id, SecureString apiKey)
```

Comment:
Create an overload that takes the full base64 encoded apiKey as well I reckon. This is what folks'll typically copy paste.

Reply:
Will set that as a `SecureString` type